### ISSUE TITLE
Textures not loading and models not exporting properly.

### ISSUE THREAD

**Issue by Hrtlss** [MrCrayfish/ModelCreator#166]
So i used to do models in this program before and texturing them without a problem. Then for a long time i stopped and now i wanted to get back to it. But when i opened one of my old models it was missing texture. No prob. i will just apply it again. Okay well long story short the texture part of the program doesnt want to load the textures. I downloaded the newest version of the program. Didnt fix it. I exported the textures again and that also didnt help. They just dont want to load and i dont know how to fix it. Esspecially that it worked before and now it doesnt even tho i didnt chage anything since last time i used it

TLDR: Texture manager used to work now it doesnt.

**Comment by mcDandy** [comment]
I have similar problem. When trying to add textures to my model texture manager is blank even if I import 
a texture 
version 0.7.0 os windows 10
was trying with vanilla 16x16 textures

**Comment by Hrtlss** [comment]
Yeah tbh i just started using different program until someone fixes the program. Also i think it might be more that just textures. I loaded a model without textures to my game and it just showed as tradicional missing texture block, no special model. so i loaded my model to different program and it showed up as a complete mess. It must be something wrong with how the information is saved. I made another model in this different program and imported it to this modelcreator and it worked like a charm so it is possible. It might be bigger issue than it seems. Maybe try doing the same thing and seeing if the same thing occurs. Im using Windows 7 64. Also i changed the name of post to include all issues

**Comment by mcDandy** [comment]
I will use version 0.4 until fixed (do not have any other version)
